### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by HassaanAhmadFarooqi**

### CODE REVIEW

File: Weatherman/.idea/workspace.xml [hunk 1]
```diff
@@ -14,20 +14,18 @@
       <file leaf-file-name="weatherman.py" pinned="false" current-in-tab="true">
```
Comment (mzulqarnain1) [inline Weatherman/.idea/workspace.xml]: Please remove this file. We do not need this.

File: Weatherman/codes/functions.py [hunk 2]
```diff
@@ -11,48 +11,26 @@ def get_date(date, what):
     date = date[first_dash + 1:]
```
Comment (mzulqarnain1) [inline Weatherman/codes/functions.py]: Please change the name of parameter "what". It should be something meaningful.

File: Weatherman/codes/functions.py [hunk 3]
```diff
@@ -11,48 +11,26 @@ def get_date(date, what):
...
     second_dash = date.find('-')
     if what == 'month':
-        month = MONTHS[int(date[:second_dash])]
-        return month
+        day = MONTHS[int(date[:second_dash])]
     elif what == 'day':
         day = date[second_dash + 1:]
         if int(day) < 10:
             day = '0' + day
```
Comment (mzulqarnain1) [inline Weatherman/codes/functions.py]: Use string formatting instead of concat.

File: Weatherman/codes/functions.py [hunk 4]
```diff
@@ -11,48 +11,26 @@ def get_date(date, what):
...
 
-def get_average(data, head_row, col, no):
+def get_average(data, head_row, col, num):
     """Calculates averages"""
     total = 0
     for k in range(head_row + 1, len(data) - 1):
         if data[k][col] != '':
             total += int(data[k][col])
-    avg = total // no
+    avg = total // num
```
Comment (mzulqarnain1) [inline Weatherman/codes/functions.py]: What is the reason for using double /

File: Weatherman/codes/weatherman.py [hunk 5]
```diff
@@ -9,7 +9,7 @@
 2nd will display given month's average
 3rd will display bar charts for whole month
 
-Pylint Score: 7.50
+Pylint Score: 6.49
```
Comment (mzulqarnain1) [inline Weatherman/codes/weatherman.py]: Use ArugmentParser Library instead of using sys.args

File: Weatherman/codes/weatherman.py [hunk 6]
```diff
@@ -9,7 +9,7 @@
...
 All objections are on my variables, pylint consider them constants
 """
```
Comment (mzulqarnain1) [inline Weatherman/codes/weatherman.py]: Maximum 3 levels of indentation are recommended. That too is a generous limit. Make your code short by making functions.

Comment (mzulqarnain1) [reply Weatherman/codes/weatherman.py]: A function should have 15 lines or less.

Comment (mzulqarnain1) [reply Weatherman/codes/weatherman.py]: Use "with open" to open files like 
`with open(file) as file_read` 
    `file_read.readline()`

File: Weatherman/codes/functions.py (hunk 2 above)
Comment (mzulqarnain1) [reply Weatherman/codes/functions.py]: Also push constants file.

File: Weatherman/codes/functions.py (hunk 4 above)
Comment (HassaanAhmadFarooqi) [reply Weatherman/codes/functions.py]: single / give division answer in float, while double in int

**Pull request closed without merging**